test/functional/test_framework/messages.py

```diff
@@ -1512,3 +1515,154 @@ class msg_no_witness_blocktxn(msg_blocktxn):
 
     def serialize(self):
         return self.block_transactions.serialize(with_witness=False)
+
+
+class msg_getcfilters:
+    __slots__ = ("filter_type", "start_height", "stop_hash")
+    command = b"getcfilters"
```

Comment:
in commit a7b375a4469fcef823a274e3f05db0f9ab7aa035 : 

Could you please squash 72784c3dac1e9d482cf5499170f689f483c1da51 into this commit?